test/quantum/normalizer_test.exs

```diff
@@ -143,10 +143,13 @@ defmodule Quantum.NormalizerTest do
   test "unnamed job as tuple with arguments" do
```

Comment:
I edited this test because it was exactly the same as on line 133
https://github.com/quantum-elixir/quantum-core/blob/663170438b234df7060c0b39ac856a501f0f05a6/test/quantum/normalizer_test.exs#L133